docs/style-guide.rst

```diff
@@ -112,6 +112,157 @@ No::
         }
     }
 
+Maximum Line Length
+===================
+
+Keeping lines under the `PEP 8 recommendation <https://www.python.org/dev/peps/pep-0008/#maximum-line-length>`_ of 79 (or 99) 
+characters helps readers easily parse the code.
+
+Wrapped lines should conform to the following guidelines.
+
+1. The first argument should not be attached to the opening parenthesis. 
+
+2. One, and only one, indent should be used.
+
+3. Each argument should fall on its own line.
+
+4. The terminating element, :code:`)` or :code:`);`, should be placed on the final line by itself.
```

Comment:
Is RST properly making this into a list if there's an empty line between the items?

Reply:
not sure.
will remove empty lines.